cookbook/setup_data/add_translation_packs.rst

```diff
@@ -0,0 +1,25 @@
+
+How to Add Translation Packs
+============================
+
```

Comment:
they are not the same
here you download translation packs, on the other page, you contribute to Crowdin
but I'll add a link here